app/src/main/java/ar/scacchipa/twittercloneapp/presentation/MainActivitViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package ar.scacchipa.twittercloneapp.presentation
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import ar.scacchipa.twittercloneapp.domain.usecase.RevokeCredentialUseCase
+import kotlinx.coroutines.launch
+
+class MainActivityViewModel (
```

Comment:
shouldn't this and the main activity be inside the main folder???

Reply:
The MainActivity and the MainActivityViewModel were moved into the mail folder.